.github/workflows/test_install_sh.yml

```diff
@@ -33,6 +33,41 @@ jobs:
         with:
           fetch-depth: 0
 
+      - id: slack
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          channel-id: ${{ env.SLACK_BOT_CHANNEL_ID }}
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#FFFF00",
+                  "blocks": [
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "<${{ env.WORKFLOW_RUN_URL }}|${{ env.WORKFLOW_RUN_NAME }}>"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Status:*\n`In Progress`"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          # C07R9GSGKEU == #mondoo-ops
+          SLACK_BOT_CHANNEL_ID: "C07R9GSGKEU"
+          WORKFLOW_RUN_URL: ${{ github.event.workflow_run.html_url }}
+          WORKFLOW_RUN_NAME: ${{ github.event.workflow_run.name }}
```

Comment:
Same here, Perhaps putting this on the job level makes life easier in the future.